crates/pallet-domains/src/staking_epoch.rs

```diff
@@ -89,6 +96,12 @@ pub(crate) fn operator_take_reward_tax_and_stake<T: Config>(
                     Some(operator) => operator,
                 };
 
+                Pallet::<T>::deposit_event(Event::OperatorRewarded {
+                    source: OperatorRewardSource::XDMProtocolFees,
+                    operator_id,
+                    reward: reward_per_operator,
+                });
```

Comment:
In most cases, `reward_per_operator` is zero, and we should not emit this event.

Reply:
We already have this event before. I just moved it here